SingularityService/src/main/java/com/hubspot/singularity/mesos/SingularityStartup.java

```diff
@@ -115,6 +129,8 @@ public void startup(MasterInfo masterInfo, SchedulerDriver driver) throws Except
   @VisibleForTesting
   void checkSchedulerForInconsistentState() {
     final long now = System.currentTimeMillis();
+    int numPendingStartupRequests = 0;
+    int totalStartupRequests = requestManager.getNumRequests();
```

Comment:
nit on naming, these are startup requests, this is just 'all' requests. Startup is specific to the type of pending request that is created in `checkActiveRequest`